arangod/RocksDBEngine/Methods/RocksDBTrxMethods.cpp

```diff
@@ -31,6 +31,8 @@
 #include "RocksDBEngine/RocksDBTransactionState.h"
 #include "Statistics/ServerStatistics.h"
 #include "Metrics/Counter.h"
+#include "Transaction/Manager.h"
+#include "Transaction/ManagerFeature.h"
```

Comment:
nit: uncommon order of include files. Are the additional includes actually still needed? I guess they are leftovers from some previous state of the PR.